test/kata5.reachDestination.test.js

```diff
@@ -1,5 +1,8 @@
 const { reachDestination } = require("../src");
 
 describe("reachDestination", () => {
-  test("returns string with estimated time of arrival", () => {});
+  test("returns string with estimated time of arrival", () => {
+    expect(reachDestination(44, 10)).toEqual("I should be there in 4.5 hours.")
+    expect(reachDestination(68, 30)).toEqual("I should be there in 2.5 hours.")
```

Comment:
I would also add a test case that is testing a result that isn't a .5 hour. eg `    expect(reachDestination(50, 25)).toEqual("I should be there in 2 hours.")`